src/components/@shared/FormInput/InputElement/AssetSelection/index.tsx

```diff
@@ -90,7 +92,11 @@ export default function AssetSelection({
                   {...props}
                   checked={selected && asset.did === selected}
                   defaultChecked={asset.checked}
-                  disabled={disabled || !asset.isAccountIdWhitelisted}
+                  disabled={
+                    isOnEditPage
+                      ? false
+                      : disabled || !asset.isAccountIdWhitelisted
+                  }
```

Comment:
issue(blocking): we should make sure the `disabled` prop gains priority. When this is set the selection should always be disabled. Only the id whitelisting needs to be overwritten on the /edit page